__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const request = require("supertest")
+const app = require("../db/app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data/index")
+const { selectTopics } = require("../models/get-api-topics.model")
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET /api/topics', () => {
+    test('responds with a 200 status code when request is successful', () => {
+
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.msg).toBe("200 - request successful");
+        });
+
+    })
+
+    test('responds with all the topics', () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.AllTopics).toEqual([
+                { slug: 'mitch', description: 'The man, the Mitch, the legend' },
```

Comment:
Currently this test is a little brittle as it is asserting the actual look of the objects in the response rather than the shape of them. It would be better to assert each object in the array has the correct properties with the correct value _type_ rather than the _exact_ value. See if you can think of a way to make the assertion forEach (😉 ) object in the array (and also make sure you protect this from return a false positive should the array be empty by asserting the length of the array before this assertion (in the same test).